llama.h

```diff
@@ -53,6 +55,9 @@ extern "C" {
         bool f16_kv;     // use fp16 for KV cache
         bool logits_all; // the llama_eval() call computes all logits, not just the last one
         bool vocab_only; // only load the vocabulary, no weights
+
+        llama_progress_handler   progress_callback; // called with a progress value between 0 and 1, pass NULL to disable
+                          void * progress_ctx;      // context pointer passed to the progress callback
```

Comment:
Let's make these consistent with the naming in `whisper.cpp`:

https://github.com/ggerganov/whisper.cpp/blob/8e361d90d7948de3ecae73e10878040044836800/whisper.h#L391-L392

I.e. `llama_progress_callback progress_callback`, etc.